test/e2e/cypress/e2e/clusters_overview.cy.js

```diff
@@ -181,3 +183,42 @@ context('Clusters Overview', () => {
     });
   });
 });
+
+describe('Forbidden action', () => {
+  beforeEach(() => {
+    cy.deleteAllUsers();
+    cy.logout();
+    const user = createUserRequestFactory.build({
+      password,
+      password_confirmation: password,
+    });
+    cy.wrap(user).as('user');
+  });
+
+  const password = 'password';
+  it('it should prevent a tag update when the user abilities are not compliant', () => {
```

Comment:
Let's put these tests under the `Tag creation` describe as in the other files

Reply:
done